shared-helpers/src/locales/es.json

```diff
@@ -557,6 +557,10 @@
   "footer.disclaimer": "Exención de responsabilidades",
   "footer.forGeneralQuestions": "Si requiere información general sobre el programa, nos puede llamar al 000-000-0000.",
   "footer.giveFeedback": "Proporcione sus comentarios",
+  "homeType.apartment": "Apartamento",
```

Comment:
All of these translations will need to be added to Chinese, Vietnamese, and Tagalag when the public site features are pulled over. but those languages don't exist in Detroit so we will need get those